testdata/project-v2/main.go

```diff
@@ -40,6 +41,7 @@ func init() {
 	_ = clientgoscheme.AddToScheme(scheme)
 
 	_ = crewv1.AddToScheme(scheme)
+	_ = v1.AddToScheme(scheme)
 	_ = corev1.AddToScheme(scheme)
```

Comment:
It has no relation, see that this update is missing in the master branch: https://github.com/kubernetes-sigs/kubebuilder/pull/1066